src/App.js

```diff
@@ -77,6 +81,22 @@ class App extends Component {
     });
   };  // End loadClient()
 
+  getPreviousDev (defaults) {
+
+    var devProps = {};
+    for (let key in defaults) {
+
+      var local = localStorage[ key ];
+      if (localStorage[ key ] !== undefined) {
+        devProps[ key ] = local === `true`;
```

Comment:
lol, because that's not the way I think about stuff. It didn't occur to me to save the `devProps` as a string as opposed to as an object, since `localStorage` doesn't like objects. I think `getItem` is async, though. Is there anything wrong with `JSON.parse(localStorage.devProps)`?